Modules/mailer/Controller/MailerController.php

```diff
@@ -56,21 +58,54 @@ public function indexAction($id_space) {
         $user = $modelUser->userAllInfo($_SESSION["id_user"]);
         $from = $user["email"];
 
+        $mails = [];
+        $mm = new Mailer();
+
+        $modelConfig = new CoreConfig();
+        $editRole = $modelConfig->getParamSpace("mailerEdit", $id_space, CoreSpace::$ADMIN);
+
+        if($this->role >= $editRole) {
+            $mails = $mm->getMails($id_space);
+        } else if ($this->role >= CoreSpace::$USER) {
+            $mails = $mm->getMails($id_space, Mailer::$SPACE_MEMBERS);
+        }
+
+        $userAppStatus = $modelUser->getStatus($_SESSION['id_user'] ?? 0);
+        $superAdmin = false;
+        if ($userAppStatus > CoreStatus::$USER) {
+            $superAdmin = true;
+        }
+
         $lang = $this->getLanguage();
-        $this->render(array("id_space" => $id_space, "lang" => $lang,
+        $this->render(array(
+            "id_space" => $id_space,
+            "lang" => $lang,
             'areasList' => $areasList,
             'resourcesList' => $resourcesList,
-            'from' => $from));
+            'from' => $from,
+            'mails' => $mails,
+            'superAdmin' => $superAdmin,
+            'role' => $this->role,
+            'editRole' => $editRole
+        ));
     }
 
     public function sendAction($id_space) {
         $this->checkAuthorizationMenuSpace("mailer", $id_space, $_SESSION["id_user"]);
+
+        $modelConfig = new CoreConfig();
+        $editRole = $modelConfig->getParamSpace("mailerEdit", $id_space, CoreSpace::$ADMIN);
+        
+        if($this->role < $editRole) {
+            throw new PfmAuthException('not enough priviledges');
```

Comment:
typo: "privileges"

Reply:
Used in feature_doc_visibility too (and I suppose in other PRs in the future). Maybe we can factorize that into a single shared method... On the other hand, it's only 2 lines...

Reply:
for single line test does not worth calling an other method, just checking role